buddyboss/buddyboss-platform
# Giphy search is not working on forums/discussion pages

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #516
Fixes #574 

### How to test the changes in this Pull Request:

1. Go to 'Forums and open a discussion and click on reply'
2. Click on 'GIF'
3. Search for any GIF
4. see, no search result.

### Proof Screenshots or Video
http://somup.com/cYn1jphQQA


### Other information:
